src/StackHCI/Az.StackHCI.psm1

```diff
@@ -131,33 +126,15 @@ $AzurePPE = "AzurePPE"
 $AzureCanary = "AzureCanary"
 
 $PortalCanarySuffix = '?feature.armendpointprefix={0}'
-$PortalAADAppPermissionUrl = '#blade/Microsoft_AAD_RegisteredApps/ApplicationMenuBlade/CallAnAPI/appId/{0}/isMSAApp/'
 $PortalHCIResourceUrl = '#@{0}/resource/subscriptions/{1}/resourceGroups/{2}/providers/Microsoft.AzureStackHCI/clusters/{3}/overview'
 
-# GUID's of the scopes generated in first party portal
-$ClusterReadPermission = "2344a320-6a09-4530-bed7-c90485b5e5e2"
-$ClusterReadWritePermission = "493bd689-9082-40db-a506-11f40b68128f"
-$ClusterNodeReadPermission = "8fa5445e-80fb-4c71-a3b1-9a16a81a1966"
-$ClusterNodeReadWritePermission = "bbe8afc9-f3ba-4955-bb5f-1cfb6960b242"
-
-# Deprecated scopes. Will be deleted during RepairRegistration
-$BillingSyncPermission = "e4359fc6-82ee-4411-9a4d-edfc7812cf24"
-$CensusSyncPermission = "8c83ab0a-0f96-40e9-940b-20cc5c5ecca9"
-
-$PermissionIds = New-Object System.Collections.Generic.List[string]
-
-$PermissionIds.Add($ClusterReadPermission)
-$PermissionIds.Add($ClusterReadWritePermission)
-$PermissionIds.Add($ClusterNodeReadPermission)
-$PermissionIds.Add($ClusterNodeReadWritePermission)
-
 $Region_EASTUSEUAP = 'eastus2euap'
 
 [hashtable] $ServiceEndpointsAzureCloud = @{
-        $Region_EASTUSEUAP = 'https://eus2euap-azurestackhci-usage.azurewebsites.net';
+        $Region_EASTUSEUAP = 'https://canary.dp.stackhci.azure.com';
         }
 
-$ServiceEndpointAzureCloudFrontDoor = "https://azurestackhci.azurefd.net"
+$ServiceEndpointAzureCloudFrontDoor = "https://dp.stackhci.azure.com"
```

Comment:
Can you also bring over the Fairfax graph endpoint update?
$GraphServiceApiScopeAzureUSGovernment = "https://graph.windows.net/.default"
should change to
$GraphServiceApiScopeAzureUSGovernment = "https://graph.microsoft.us/.default"